core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -255,12 +255,12 @@ object KafkaConfig {
   val LogFlushIntervalMsProp = "log.flush.interval.ms"
   val LogFlushOffsetCheckpointIntervalMsProp = "log.flush.offset.checkpoint.interval.ms"
   val LogPreAllocateProp = "log.preallocate"
-  val MessageFormatVersionProp = "message.format.version"
+  val LogMessageFormatVersionProp = "log.message.format.version"
```

Comment:
@gwenshap Good idea. That is probably less error prone. I will update the patch.